File: beginner/1046.cpp
#include <stdio.h>
 
int main() {
	int start, end, hours;
	
	scanf("%d %d", &start, &end);
	
	if(start > end) {
		hours = (24 - start) + end;
	} else if(start < end) {
		hours = end - start;
	} else {
		hours = 24;
	}
 	
	printf("O JOGO DUROU %d HORA(S)\n", hours);
    return 0;
}

File: C#/begginer/1046.cs
using System; 

class URI {

  static void Main(string[] args) {
    int[] n = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
    int time;
    
    if(n[0] > n[1]) {
      time = 24 - (n[0] - n[1]);
    } else if(n[0] < n[1]) {
      time = n[1] - n[0];
    } else {
      time = 24;
    }

    Console.WriteLine($"O JOGO DUROU {time} HORA(S)");
  }

}